xen/arch/x86/tpm.c

```diff
@@ -34,12 +34,22 @@ asm (
 #endif
 #define __va(x)     _p(x)
 
+void *memset(void *dest, int c, size_t n)
```

Comment:
Isn't any memset available in Xen?

Reply:
It's not available when this file is compiled separately with `__EARLY_TPM__`.

Reply:
I see.